test/data/s3/test_s3.py

```diff
@@ -351,6 +389,13 @@ def test_init_options(s3root, pathspecs, expected):
         assert_results(s3objs, expected)
         assert_results(s3.get_all(), expected, info_should_be_empty=True)
 
+    # option 5) run object
+    if DO_TEST_RUN:
```

Comment:
What's the intention behind this conditional?

Reply:
Test does not work in OSS (works internally). I merged all test files for S3 now so this was one of the differences and this seemed like an easy enough way to add it without being too problematic. I can add a comment if you want.